.github/workflows/test-azureblob.yml

```diff
@@ -15,18 +15,17 @@ jobs:
         strategy:
             matrix:
                 go-version:
-                    - 1.19.x
-                    - 1.20.x
-                    - 1.21.x
+                    - 1.22.x
```

Comment:
May as well remove 1.22, it's End Of Life

Reply:
Should I remove it elsewhere testcontainers is used?

Reply:
Yes, only 1.23 and 1.24 are getting updates from the Go team

Reply:
In a follow-up PR, should we update the benchmarks version too?

Reply:
I forgot why we did 1.21, it was prob latest at that point, or its the one listed in the main go.mod file